landingpage/index.html

```diff
@@ -0,0 +1,152 @@
+                    <h1>Admission Staff</h1>
+                    <p>Looking to manage millions of applications? Start now.</p>
+{#                    <p>#}
+                    <a href="{% url 'logIn:login_user' constant_ut_adminStaff %}"><button class="btn btn-lg btn-primary">Begin Here</button></a>
+{#                    <a class="btn btn-lg btn-primary" onclick="location.href={% url 'logIn:login_user' 1 %}" role="button">Begin here</a>#}
+{#                    </p>#}
+                </div>
+            </div>
+        </div>
+    </div>
+    <a class="carousel-control-prev" href="#myCarousel" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#myCarousel" role="button" data-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+    </a>
+</div>
+
+
+<!-- Marketing messaging and featurettes
+================================================== -->
+<!-- Wrap the rest of the page in another container to center all the content. -->
+
+<div class="container marketing">
+
+    <!-- Three columns of text below the carousel -->
+
+
+    <!-- START THE FEATURETTES -->
+
+    <!--<hr class="featurette-divider">-->
+
+    <div class="row featurette">
+        <div class="col-md-7">
+            <h2 class="featurette-heading">Student <span class="text-muted"></span></h2>
+            <p class="lead">As a student, you will be able to search for high schools using various filters such as
+                location, safety, programs offered, school performance and much more. By filling out a simple
+                application form, you will be able to apply to as many schools as you wish to with just one click of a
+                button.</p>
+        </div>
+        <div class="col-md-5">
+            <img class="featurette-image img-fluid mx-auto" src="{% static '/landingpage/students.jpeg' %}"
+                 alt="Generic placeholder image">
+        </div>
+    </div>
+
+    <hr class="featurette-divider">
+
+    <div class="row featurette">
+        <div class="col-md-7">
+            <h2 class="featurette-heading">Admission Staff<span class="text-muted"></span></h2>
+            <p class="lead">As an admissions staff of a high school, OneApply provides you with myriad of features to
+                view, sort and manage millions of applications. Sit back and relax while OneApply does everything for
+                you from managing applications to updating the number of seats available to enroll students.</p>
+        </div>
+        <div class="col-md-5">
+            <img class="featurette-image img-fluid mx-auto" src="{% static '/landingpage/teacher.jpeg' %}"
+                 alt="Generic placeholder image">
+        </div>
+    </div>
+
+    <!--<hr class="featurette-divider">-->
```

Comment:
remove the commented code if not being used.

Reply:
solved